docs/UserGuide.md

```diff
@@ -156,105 +156,150 @@ Examples:
+  
+* #### 3.6.2 Find items with partial name match
+  Format: `find */KEYWORD [MORE_KEYWORDS]`
+  <div markdown="block" class="alert alert-info">
 
-Examples:
+  **:information_source: Notes about the command:**<br>
+  
+  * Partial words will be matched e.g. `Choco` will match `Chocolates`<br></br>
+  
+  </div>
+  
+  Examples:
+  * `find */Burger` returns `CheeseBurger` and `fishburger`
+  * `find */cheese egg` returns `MacAndCheese` and `eggs`
 
-* `find Chocolate` followed by `sort quantity` sorts the items from the result of the `find` command.
-* `list Room 2` followed by `sort expiryDate` sorts the recorded list of items in Room 2 by expiry date.
 
-### Clearing all entries : `clear`
+### 3.7. Viewing expiring items : `reminder`  <a name = "reminder"></a>
 
-Clears all entries from the location book.
+Want to know which items of yours are expiring soon? Then, this is the right command for you.<br>
 
-Format: `clear`
+Format: `reminder number TIME_UNIT`
+<div markdown="block" class="alert alert-info">
 
-### Exiting the program : `exit`
+**:information_source: Notes about the reminder command:**<br>
 
-Exits the program.
+* `TIME_UNIT` must be either `days` or `weeks`.
+*  Items without expiry date will not be shown.
+*  `number` **must be a positive integer** 1, 2 , 3, …​
 
-Format: `exit`
+</div>
 
-### Saving the data
+Examples:
+* `reminder 7 days` returns a list containing all items that are expiring within the next 7 days.
+* `reminder 2 weeks` returns a list containing all items that are expiring within the next 2 weeks.
+* Given today is 10 March 2021, and the inventory has 2 items: chocolate with expiry date of 13 March 2021 and 
+  banana with an expiry date of 20 March 2021.
+  * `reminder 5 days` returns the chocolate only.
+  * `reminder 2 weeks` returns both the chocolate and the banana.
 
-StoreMando data are saved in the hard disk automatically after any command that changes the data. There is no need to
-save manually.
+### 3.8. Sorting items : `sort` <a name = "sort"></a>
 
-### Editing the data file
+Want to see which items you are running out of or which items you have to consume soon? Use this command to find out!
 
-StoreMando data are saved as a JSON file `[JAR file location]/data/storemando.json`. Advanced users are welcome to
-update data directly by editing that data file.
+You can use this command in 2 different ways.
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, StoreMando will discard all data and start with an empty data file at the next run.
-</div>
+* #### 3.8.1. Sorting items by quantity
+  Format 1: `sort quantity`
+  <br></br>
+  Example:
+  * `sort quantity` sorts the items in the displayed list in ascending order of quantity.
 
-### Archiving data files `[coming in v2.0]`
+* #### 3.8.2. Sorting items by expiry date
+  Format 2: `sort expiryDate`
```

Comment:
also this, it should be expirydate, emphasise to the users too that it is case insensitive. :)

Reply:
alright will do so!